src/main/java/com/scalar/db/storage/dynamo/SelectStatementHandler.java

```diff
@@ -0,0 +1,267 @@
+package com.scalar.db.storage.dynamo;
+
+import com.scalar.db.api.Consistency;
+import com.scalar.db.api.Get;
+import com.scalar.db.api.Operation;
+import com.scalar.db.api.Scan;
+import com.scalar.db.exception.storage.ExecutionException;
+import com.scalar.db.io.Value;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.IntStream;
+import javax.annotation.concurrent.ThreadSafe;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
+import software.amazon.awssdk.services.dynamodb.model.DynamoDbException;
+import software.amazon.awssdk.services.dynamodb.model.GetItemRequest;
+import software.amazon.awssdk.services.dynamodb.model.GetItemResponse;
+import software.amazon.awssdk.services.dynamodb.model.QueryRequest;
+import software.amazon.awssdk.services.dynamodb.model.QueryResponse;
+
+/**
+ * A handler class for select statement
+ *
+ * @author Yuji Ito
+ */
+@ThreadSafe
+public class SelectStatementHandler extends StatementHandler {
```

Comment:
Just a question. If the secondary index is used, can this `SelectStatementHandler` be used in the same way?

Reply:
I think it can be used in the same way because we can use `client.query()` API for the secondary index too.